### ISSUE TITLE
r

### ISSUE THREAD

**Issue by sergey-smolin** [sergey-smolin/TestRepo#10]
adf

**Comment by hisumai** [comment]
Hello,
Does "ADF" mean Augmented Dickey Fuller Test or something else?

**Comment by sergey-smolin** [comment]
Yeah, and it gets memorized really well and it's also fast to type

**Comment by hisumai** [comment]
Hello, it has been quite a while since my last comment.

So, have you got what you were looking for regarding the ADF function in R?

If not, can you please inform whether you want to create a new ADF Function in R and publish it in the list of R packages, even though, ADF functionality is present in the form of adf.test function?
